### PR TITLE
New Datasource: `github_collaborators`

### PR DESCRIPTION
**Pull request by megan07**
Adding collaborators datasource. Addresses #103 
```
=== RUN   TestAccGithubCollaboratorsDataSource_basic
--- PASS: TestAccGithubCollaboratorsDataSource_basic (6.13s)
```

### CODE REVIEW

File: github/data_source_github_collaborators.go [hunk 1]
```diff
@@ -0,0 +1,194 @@
...
+	}
+
+	d.SetId(fmt.Sprintf("%s/%s/%s", owner, repo, affiliation))
+	d.Set("owner", owner)
+	d.Set("repository", repo)
+	d.Set("affiliation", affiliation)
+
+	totalCollaborators := make([]interface{}, 0)
+	for {
+		collaborators, resp, err := client.Repositories.ListCollaborators(ctx,
```
Comment (katbyte) [inline github/data_source_github_collaborators.go]: Maybe merge these two lines?
```suggestion
		collaborators, resp, err := client.Repositories.ListCollaborators(ctx, owner, repo, options)
```

File: github/data_source_github_collaborators.go [hunk 2]
```diff
@@ -0,0 +1,194 @@
...
+
+		Schema: map[string]*schema.Schema{
+			"owner": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
+			"repository": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
```
Comment (katbyte) [inline github/data_source_github_collaborators.go]: Should we validate that this is not empty?

File: github/data_source_github_collaborators.go [hunk 3]
```diff
@@ -0,0 +1,194 @@
...
+
+func dataSourceGithubCollaborators() *schema.Resource {
+	return &schema.Resource{
+		Read: dataSourceGithubCollaboratorsRead,
+
+		Schema: map[string]*schema.Schema{
+			"owner": {
+				Type:     schema.TypeString,
+				Required: true,
+			},
```
Comment (katbyte) [inline github/data_source_github_collaborators.go]: Should we validate that this is not empty?

File: github/data_source_github_collaborators.go [hunk 4]
```diff
@@ -0,0 +1,194 @@
+package github
+
+import (
+	"context"
+	"fmt"
+	"github.com/hashicorp/terraform/helper/validation"
```
Comment (katbyte) [inline github/data_source_github_collaborators.go]: I would run goimports on this file

File: github/data_source_github_collaborators_test.go [hunk 5]
```diff
@@ -0,0 +1,42 @@
+package github
+
+import (
+	"fmt"
+	"github.com/hashicorp/terraform/helper/acctest"
```
Comment (katbyte) [inline github/data_source_github_collaborators_test.go]: I would run goimports on this file

**Comment by megan07** [comment]
closing down in favor of #239

**Pull request closed without merging**